### PR TITLE
[Backend] Create main schema

### PR DESCRIPTION
**Pull request by KE-CY**
與這支 PR 有關聯的 Asana 卡片:

URL: 
[DB] Tag Schema(https://app.asana.com/0/1201219031105843/1201244929027378/f)
[DB] ProductTag Schema(https://app.asana.com/0/1201219031105843/1201219341450394/f)
[DB] Product Schema(https://app.asana.com/0/1201219031105843/1201219341450392/f)
[DB] Main Schema(https://app.asana.com/0/1201219031105843/1201224244424760/f)
[DB] OrderProduct Schema(https://app.asana.com/0/1201219031105843/1201219341450396/f)
[DB] Order Schema(https://app.asana.com/0/1201219031105843/1201219341450398/f)
[DB] Tag Schema(https://app.asana.com/0/1201219031105843/1201234678753305/f)
# Description

**新增 Tag ,ProductTag ,Product ,Order ,OrderProduct ,Handover Schema**

# Screenshot or more (Optional)
image
image
image
image
image
image
image

-----------------

提交 PR 前，請先檢查以下是否已完成：

* [ ✔️] 該分支是從最新 `master` 切出來 (如果不是，請先 rebase/merge 它)
* [❌ ] 是否新增測試，或被現有的測試覆蓋
* [✔️ ] 這支 PR 是否有明確的標題、內容描述

-----------------

補充說明:

### CODE REVIEW

File: server/src/entity/tag.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import {
+    Entity,
+    PrimaryGeneratedColumn,
+    Column,
+    CreateDateColumn,
+    UpdateDateColumn,
+    DeleteDateColumn,
+    BaseEntity,
+} from "typeorm";
+@Entity()
```
Comment (p0937507934) [inline server/src/entity/tag.ts:10]: Need a blank between line 9 to line 10 ?

**Comment by p0937507934** [comment]
Need to resolve conflicts because new feature into main.

**Comment by KE-CY** [comment]
The latest main has been merged

File: server/src/entity/tag.ts (hunk 1 above)
Comment (KE-CY) [reply server/src/entity/tag.ts:11]: Changes have been made

File: server/src/entity/handover.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    PrimaryGeneratedColumn,
+    Column,
+    CreateDateColumn,
+    UpdateDateColumn,
+    DeleteDateColumn,
+    OneToOne,
+    JoinColumn,
+    BaseEntity,
+} from "typeorm";
+import { User } from "./user";
```
Comment (p0937507934) [inline server/src/entity/handover.ts:12]: these lines also need to modify.

Comment (KE-CY) [reply server/src/entity/handover.ts:12]: Completed with the update, and also updated other files.

File: server/src/entity/orderProduct.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    Column,
+    CreateDateColumn,
+    UpdateDateColumn,
+    DeleteDateColumn,
+    OneToOne,
+    JoinColumn,
+    ManyToOne,
+} from "typeorm";
+import { Order } from "./order";
+import { Product } from "./product";
```
Comment (p0937507934) [inline server/src/entity/orderProduct.ts:13]: blank

File: server/src/entity/order.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    PrimaryGeneratedColumn,
+    Column,
+    CreateDateColumn,
+    UpdateDateColumn,
+    DeleteDateColumn,
+    JoinColumn,
+    BaseEntity,
+    ManyToOne
+} from "typeorm";
+import { Store } from "./store";
```
Comment (p0937507934) [inline server/src/entity/order.ts:12]: blank

File: server/src/entity/handover.ts (hunk 2 above)
Comment (p0937507934) [reply server/src/entity/handover.ts:12]: not seen any chaged.

File: server/tsconfig.json [hunk 5]
```diff
@@ -26,7 +26,7 @@
     /* Modules */
     "module": "commonjs" /* Specify what module code is generated. */,
     "rootDir": "./" /* Specify the root folder within your source files. */,
-    // "moduleResolution": "node",                       /* Specify how TypeScript looks up a file from a given module specifier. */
+    "moduleResolution": "node",                       /* Specify how TypeScript looks up a file from a given module specifier. */
```
Comment (p0937507934) [inline server/tsconfig.json:29]: what is this config do?

File: server/package.json [hunk 6]
```diff
@@ -9,7 +9,8 @@
     "build": "tsc",
     "start": "npm run build && node ./src/dist/src/index.js",
     "eslint": "eslint --fix ./src/**.ts",
-    "prettier": "sudo prettier --write --ignore-unknown ./src/**.ts "
+    "prettier": "prettier --write --ignore-unknown",
+    "linux_prettier": "sudo prettier --write --ignore-unknown ./src/**.ts "
```
Comment (p0937507934) [inline server/package.json:13]: why add this script?

**Review by p0937507934 — APPROVED** [review]

**Pull request merged**